src/include/migraphx/op/where.hpp

```diff
@@ -42,9 +42,19 @@ struct where
 
     shape compute_shape(std::vector<shape> inputs) const
     {
-        check_shapes{inputs, *this}.has(3).same_dims();
+        check_shapes{inputs, *this, true}.has(3).same_dims();
         auto s1 = inputs.at(1);
         auto s2 = inputs.at(2);
+        if(s1.dynamic() or s2.dynamic())
+        {
+            if(s1 == s2)
+                return s1;
+            // MIGRAPHX_THROW("BINARY: " + point_function() + ": fixed-dyn shape for inputs");  what
+            // is point_function() for?
+            MIGRAPHX_THROW("WHERE: fixed-dyn shape for inputs");
```

Comment:
Can you change this throw message ? It doesn't seem helpful. 
IMO. It should be more related to  "shape mismatch"

Reply:
I'll reword the message